netbox/templates/login.html

```diff
@@ -11,8 +11,8 @@
 
       {# NetBox logo #}
       <div class="text-center mb-4">
-        <img src="{% static 'logo_netbox_dark_teal.svg' %}" alt="{% trans "NetBox Logo" %}" height="80" class="hide-theme-dark">
-        <img src="{% static 'logo_netbox_bright_teal.svg' %}" alt="{% trans "NetBox Logo" %}" height="80" class="hide-theme-light">
+        <img src="{% static 'logo_netbox_dark_teal.svg' %}" alt="{% trans "NetBox Logo" %}" height="80" class="hide-theme-dark" style="height: 80px;">
```

Comment:
We should avoid setting absolute CSS parameters directly on individual elements wherever possible. Let's instead introduce `logo` and `sso-icon` classes for the affected elements, which can be reused and/or tweaked easily if needed.